### DISCUSSION TITLE
4DC87FCC-60DF-47B3-850B-81FE3D74DD70

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#52] [General]
# 4DC87FCC-60DF-47B3-850B-81FE3D74DD70

K2-39, is a K-type subgiant star about 1,008 light-years away in the constellation Aquarius. A super-neptunian planet nearly the size of Saturn completes an single orbit once every 4.6 days at a distance of 9.52 million km from its parent star. The planet is predicted to be destroyed by orbital decay in the next 150 million years

https://celestia.mobi/addon?item=4DC87FCC-60DF-47B3-850B-81FE3D74DD70&lang=en

**Comment by exocritic5457** [comment]
Whew, good thing something that neon green is getting destroyed. Albeit, eventually.
